packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,279 @@
+  font-size: 12px;
+  position: relative;
+
+  &:focus {
+    outline: none;
+  }
+`;
+
+const menuNameStyle = css`
+  margin-right: 2px;
+  margin-left: 2px;
+`;
+
+const activeMenuButtonStyle = css`
+  background-color: ${uiColors.gray.light2};
+`;
+
+const nameStyle = css`
+  font-size: 16px;
+  color: ${uiColors.gray.dark2};
+  font-weight: bold;
+  margin: 4px 0px 2px;
+`;
+
+const truncate = css`
+  width: 162px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const closedIconStyle = css`
+  transform: rotate(180deg);
+  transition: color 200ms ease-in-out;
+  color: ${uiColors.gray.base};
+`;
+
+const openIconStyle = css`
+  margin-top: 2px;
+  color: ${uiColors.gray.base};
+`;
+
+const accountMenuGroupStyle = css`
+  padding: 20px 20px 14px;
+`;
+
+const accountButtonStyle = css`
+  margin-top: 12px;
+  width: 100%;
+  display: inline-flex;
+  justify-content: center;
+`;
+
+const descriptionStyle = css`
+  margin: 0px;
+  font-size: 12px;
+  color: ${uiColors.gray.dark1};
+  text-decoration: none;
+`;
+
+const logoutContainerHeight = css`
+  height: 46px;
+`;
+
+const menuItems = [
+  {
+    displayName: 'Atlas',
+    description: 'cloud.mongodb.com',
+    href: 'https://cloud.mongodb.com',
+    slug: 'atlas',
+  },
+  {
+    displayName: 'University',
+    description: 'university.mongodb.com',
+    href: 'https://university.mongodb.com',
+    slug: 'university',
+  },
+  {
+    displayName: 'Cloud Support',
+    description: 'support.mongodb.com',
+    href: 'https://support.mongodb.com',
+    slug: 'support',
+  },
+] as const;
+
+export const Products = {
+  Atlas: 'atlas',
+  University: 'university',
+  Support: 'support',
+} as const;
+
+type Products = typeof Products[keyof typeof Products];
+
+interface SSOMenuProps {
+  /**
+   * Object that contains information about the active user. {name: 'string', email: 'string'}
+   */
+  user: { name: string; email: string };
+
+  /**
+   * MongoDB product that is currently active: ['atlas', 'university', 'support'].
+   */
+  activeProduct: Products;
+
+  /**
+   * Callback invoked after the user clicks log out.
+   */
+  onLogout?: React.MouseEventHandler;
+
+  /**
+   * Callback invoked after the user clicks a product.
+   */
+  onProductChange?: React.MouseEventHandler;
+}
+
+/**
+ * # SSOMenu
+ *
+ * ```
+<SSOMenu
+    user={{ name: 'Alex Smith', email: 'alex.smith@youwork.com' }}
+    activeProduct="atlas"
+    onLogout={() => console.log('On logout')}
+    onProductChange={() => console.log('Switching products')}
+  />
+ * ```
+ * @param props.user Object that contains information about the active user. {name: 'string', email: 'string'}
+ * @param props.activeProduct  MongoDB product that is currently active: ['atlas', 'university', 'support'].
+ * @param props.onLogout Callback invoked after the user clicks log out.
+ * @param props.onProductChange Callback invoked after the user clicks a product.
+ *
+ */
+function SSOMenu({
```

Comment:
Talked in Slack, but let's rename this to something that has a longer shelf life. MongoDBMenu or something similar.